src/CosmosEditorManager.ts

```diff
@@ -81,11 +87,30 @@ export class CosmosEditorManager implements vscode.Disposable {
         }
     }
 
+    private async loadPersistedEditor(documentUri: vscode.Uri, tree: AzureTreeDataProvider): Promise<string> {
+        const pickledLabels = this._globalState.get(this._editorConfigs);
+        //Based on the documentUri, split just the appropriate key's value on '/'
+        const relevantFilePath = Object.keys(pickledLabels).find((label) => path.relative(documentUri.fsPath, label) === '');
+        if (relevantFilePath) {
+            const editorNode = await tree.findNode(pickledLabels[relevantFilePath]);
+            let editor: ICosmosEditor;
```

Comment:
You have to check if 'editorNode' is defined. It's possible the user logged out and the node can't be found